t/test_postgresql

```diff
@@ -1,16 +1,20 @@
 #!/bin/sh -e
 
+KEEP_DB="${KEEP_DB:-0}"
 DIR=$1
 if test -z "$DIR"; then
     DIR=$(mktemp -d)
 else
     DIR=$(readlink -f "$DIR")
-fi
-if test -e "$DIR"/postmaster.pid; then
-  pg_ctl -D "$DIR" stop
-fi
-if test -d "$DIR"; then
-  rm -r "$DIR"
+    if test "$KEEP_DB" = "0"; then
+        if test -d "$DIR"; then
+            test -e "$DIR"/postmaster.pid && pg_ctl -D "$DIR" stop
+            rm -r "$DIR"
+        fi
+    else
+        echo "Database in $DIR will be re-used because KEEP_DB is set to 1."
```

Comment:
```suggestion
        echo "Reusing existing database in $DIR because KEEP_DB is set to 1."
```

would read a bit less dangerous. The sentence "database will be re-used" sounds like more bad things could happen as in. "we warned you, this will be going bad" ;)

Reply:
Well... bad things did happen to Tina but fine by me